src/utils/format-numbers.js

```diff
@@ -53,14 +53,28 @@ export function formatPercent(value: number): string {
 }
 
 export function formatBytes(bytes: number): string {
-  if (bytes < 4 * 1024) {
+  if (bytes < 10000) {
+    // Use singles up to 10,000.  I think 9,360B looks nicer than 9.36KB.
     return formatNumber(bytes) + 'B';
-  } else if (bytes < 4 * 1024 * 1024) {
-    return formatNumber(bytes / 1024) + 'KB';
-  } else if (bytes < 4 * 1024 * 1024 * 1024) {
-    return formatNumber(bytes / (1024 * 1024)) + 'MB';
+  } else if (bytes < 1024 * 1024) {
+    return formatNumber(bytes / 1024, 3, 2) + 'KB';
+  } else if (bytes < 1024 * 1024 * 1024, 3, 2) {
```

Comment:
oops, the condition is wrong here :) (spotted by eslint !)

Reply:
Thanks.